app/config/locale/translations/ms.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "ms",
     "settings.direction": "ltr",
     "emails.sender": "%s Team",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Pengesahan Akaun",
+    "emails.verification.hello": "Hey {{name}}",
+    "emails.verification.body": "Tekan pautan ini untuk mengesahkan alamat email anda.",
+    "emails.verification.footer": "Sekiranya anda tidak membuat permintaan untuk mengesahkan email ini, sila abaikan mesej ini.",
+    "emails.verification.thanks": "Terima kasih",
+    "emails.verification.signature": "{{project}} team",
+    "emails.magicSession.subject": "Log masuk",
+    "emails.magicSession.hello": "Hey",
```

Comment:
https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L13

Please fix the punctuation as in the English translation file